Description: Add missing ldflags & cxxflags of zlib
 - zlib's deflate() is used in zen/zlib_wrap.*
 .
 - update compilation flags:
    - CXXFLAGS: add flags for zlib
    - LDFLAGS: add flags for zlib
 .
 Since ffs calls the deflate function of zlib in zen/zlib_wrap.*, we have to
 add -lz.
 .
 Otherwise build would fail this way:
  /usr/bin/ld: /tmp/FreeFileSync_Make/ffs/src/../../zen/zlib_wrap.cpp.o: undefined reference to symbol 'deflate'
  /usr/bin/ld: /lib/x86_64-linux-gnu/libz.so.1: error adding symbols: DSO missing from command line
 .
 The RealTimeSunc binary doesn't use zen/zlib_wrap.*, so it is not required in
 that case.
 .
 Required since 10.0
 .
Author: Fab Stz <fabstz-it@yahoo.fr>
Last-Modified: 2024-02-07
Forwarded: https://freefilesync.org/forum/viewtopic.php?p=42392

--- a/FreeFileSync/Source/Makefile
+++ b/FreeFileSync/Source/Makefile
@@ -32,6 +32,9 @@
 LDFLAGS += `$(PKG_CONFIG) --libs libselinux`
 endif
 
+CXXFLAGS  += `$(PKG_CONFIG) --cflags zlib`
+LDFLAGS += `$(PKG_CONFIG) --libs zlib`
+
 cppFiles=
 cppFiles+=application.cpp
 cppFiles+=base_tools.cpp
